### PR TITLE
media-tv/mythtv: version bump to 32.0

### PR DESCRIPTION
**Pull request by thebitpit**
Closes: https://bugs.gentoo.org/837659
Package-Manager: Portage-3.0.30, Repoman-3.0.3
Signed-off-by: Wilson Michaels <thebitpit@austincustomerrands.com>

### CODE REVIEW

**Comment by thebitpit** [comment]
Please review. 
Tested with all USE flags, no USE flags, USE flags I use on my production system. Upgrade worked on my production system.

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,427 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8,9} )
+
+inherit eutils flag-o-matic python-any-r1 qmake-utils readme.gentoo-r1 systemd user-info
+
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:9]: Eutils should be droppable, mah need to replace with split eclasses depending on functionality used

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,427 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:4]: 8?

Comment (thebitpit) [reply media-tv/mythtv/mythtv-32.0.ebuild:4]: EAPI=8 works with no modifications to ebuild.

Comment (thesamesam) [reply media-tv/mythtv/mythtv-32.0.ebuild:4]: Please force push with these changes

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,427 @@
...
+	if use debug; then
+		myconf+=(--compile-type=debug)
+		myconf+=(--disable-stripping) # FIXME: does not disable for all files, only for some
+		myconf+=(--enable-valgrind) # disables timeouts for valgrind memory debugging
+	else
+		myconf+=(--compile-type=release)
+	fi
+
+	# Build boosters
+	has ccache "${FEATURES}" || myconf+=(--disable-ccache)
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:240]: These two actually seem quite suspicious and shouldn't be used. Enabling ccache and distcc are done outside of ebuild and set via env appropriately.

Comment (thesamesam) [reply media-tv/mythtv/mythtv-32.0.ebuild:240]: But we can fix this next time given this fixes dav1d.

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,427 @@
...
+
+	# CPU settings
+	# Mythtv's configure is borrowed from ffmpeg,
+	# Bug #172723
+	# Try to get cpu type based on CFLAGS.
+	# We need to do this so that features of that CPU will be better used
+	# If they contain an unknown CPU it will not hurt since ffmpeg's configure
+	# will just ignore it.
+	for i in $(get-flag march) $(get-flag mcpu) $(get-flag mtune) ; do
+		[ "${i}" = "native" ] && i="host" # bug #273421
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:251]: bash tests please ([[ ]]), but fix next time

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,427 @@
...
+	has distcc "${FEATURES}" || myconf+=(--disable-distcc)
+
+	# CPU settings
+	# Mythtv's configure is borrowed from ffmpeg,
+	# Bug #172723
+	# Try to get cpu type based on CFLAGS.
+	# We need to do this so that features of that CPU will be better used
+	# If they contain an unknown CPU it will not hurt since ffmpeg's configure
+	# will just ignore it.
+	for i in $(get-flag march) $(get-flag mcpu) $(get-flag mtune) ; do
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:250]: put 'local i' above

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,427 @@
...
+		--prefix="${EPREFIX}/usr" \
+		--cc="$(tc-getCC)" \
+		--cxx="$(tc-getCXX)" \
+		--ar="$(tc-getAR)" \
+		--optflags="${CFLAGS}" \
+		--extra-cflags="${CFLAGS}" \
+		--extra-cxxflags="${CXXFLAGS}" \
+		--extra-ldflags="${LDFLAGS}" \
+		--qmake=$(qt5_get_bindir)/qmake \
+		"${myconf[@]}"
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:351]: missing || die!

File: media-tv/mythtv/mythtv-32.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,427 @@
...
+
+	# Make shell & perl scripts executable
+	find "${ED}" -type f \( -name '*.sh' -o -name '*.pl' \) -exec chmod a+x {} \; || die "Failed to make script executable"
+}
+
+pkg_postinst() {
+	readme.gentoo_print_elog
+}
+
+pkg_info() {
```
Comment (thesamesam) [inline media-tv/mythtv/mythtv-32.0.ebuild:419]: Why is this here? We can just drop?

File: media-tv/mythtv/mythtv-32.0.ebuild (hunk 6 above)
Comment (thebitpit) [reply media-tv/mythtv/mythtv-32.0.ebuild:351]: OK, I'll fix that.

File: media-tv/mythtv/mythtv-32.0.ebuild (hunk 7 above)
Comment (thebitpit) [reply media-tv/mythtv/mythtv-32.0.ebuild:419]: Yes. Drop it.

File: media-tv/mythtv/mythtv-32.0.ebuild (hunk 5 above)
Comment (thebitpit) [reply media-tv/mythtv/mythtv-32.0.ebuild:250]: Done

File: media-tv/mythtv/mythtv-32.0.ebuild (hunk 3 above)
Comment (thebitpit) [reply media-tv/mythtv/mythtv-32.0.ebuild:240]: OK
